guides/document_lifecycle.md

```diff
@@ -15,7 +15,7 @@ Next, create an index named `movies` with the default settings:
 ```javascript 
 const index = 'movies';
 client.indices.exists({ index: index })
-  .then(exists => {
+  .then({ body: exists } => {
     if (!exists) {
```

Comment:
I think we should simply access the body of the response (named exists) to get the boolean value:
`if(!exists.body) {`

Reply:
Then the overall fix would be:
```
client.indices.exists({ index: index })
  .then(exists => {
  if (!exists.body) {
    return client.indices.create({ index: index });
  }
}).then(response => console.log(`Index ${response.body.index} created`));
```

Reply:
Okay! Made the change :)